test/e2e/secret.py

```diff
@@ -0,0 +1,18 @@
+import boto3
```

Comment:
We typically create a `*Validator` class, and pass the boto3 client as a param - https://github.com/aws-controllers-k8s/ec2-controller/blob/main/test/e2e/tests/helper.py#L20

That's because we don't want to initialise the boto3 client in too many different places, since we pass configuration overrides to it sometimes

Reply:
refactor [refactor test](https://github.com/aws-controllers-k8s/secretsmanager-controller/pull/2/commits/226847434434b9b4e68bc2f5d5f77779d23dda5b)